Perl: Another round of mulitple updates

peterbud opened the pull request:
New package: perl-spiffy (version 0.46)
New package: perl-test-base (Version: 0.89)
New package: perl-Test-YAML (version: 1.07)
New package: perl-Test-NoWarnings (Version: 1.04)
New package: perl-Test-Deep (Version: 1.128)
perl-File-Copy-Recursive: Update to 0.44
perl-net-http: Update to 6.18
perl-io-socket-ssl: Update to 2.056
perl-YAML: Update to 1.26
perl-Error: Update to 0.17026
perl-HTTP-Message: Update to 6.18
perl-ack: Update to 2.24
perl-Net-DNS: Update to 1.16
perl-Exporter-Tiny: Update to 1.002001

piotrvv commented:
The perl-Test-Tester module has a conflict with the perl
error: transaction failed (conflicting files)
perl-Test-Tester: /usr/share/man/man3/Test.Tester.3pm.gz occurs in the file system (the owner is perl-Test-Simple)
perl-Test-Tester: /usr/share/man/man3/Test.Tester.Capture.3pm.gz appears on the file system (the owner is perl-Test-Simple)
perl-Test-Tester: /usr/share/man/man3/Test.Tester.CaptureRunner.3pm.gz appears on the file system (the owner is perl-Test-Simple)
perl-Test-Tester: /usr/share/perl5/vendor_perl/Test/Tester.pm occurs in the file system (the owner is perl-Test-Simple)
perl-Test-Tester: /usr/share/perl5/vendor_perl/Test/Tester/Capture.pm appears in the file system (the owner is perl-Test-Simple)
perl-Test-Tester: /usr/share/perl5/vendor_perl/Test/Tester/CaptureRunner.pm appears in the file system (the owner is perl-Test-Simple)
perl-Test-Tester: /usr/share/perl5/vendor_perl/Test/Tester/Delegate.pm occurs in the file system (the owner is perl-Test-Simple)
Errors occurred, no package updated.

peterbud commented:
@piotrvv : Thanks for spotting this. Indeed `perl-Test-Tester` is already included with `perl-Test-Simple`. I have modified the two dependent package to have dependency on `perl-Test-Simple`, and removed the `perl-Test-Tester`, as it is redundant. 

Interestingly there is no `perl-Test-Simple` in arch repo.

piotrvv commented:
I'm glad that I could help.
